front/src/service.ts

```diff
@@ -420,4 +421,71 @@ export class ScratchyService {
         this.messageCache.delete(messageID);
         return response.data;
     }
-}
\ No newline at end of file
+
+/* Call server with Websocket */
+
+     /**
+     *
+     * @async
+     * @augments ScratchyService
+     * @param {Room} room - affected room
+     * @returns {Promise<undefined>} -
+     */
+    async joinRoomWebsocket(room: Room): Promise<void> {
+        socket.on("join", (socket) => {  
+            socket.join(room.id);
+        });
+    }
+
+     /**
+     *
+     * @async
+     * @augments ScratchyService
+     * @param {Room} room - affected room
+     * @returns {Promise<undefined>} -
+     */
+      async quitRoomWebsocket(room: Room): Promise<void> {
+        socket.on("quit", (socket) => {  
+            socket.leave(room.id);
+        });
+    }
+
+    /**
+     *
+     * @async
+     * @augments ScratchyService
+     * @returns {Promise<Message>} - message data
+     */
+     async newMessageWebsocket(): Promise<void> {
+        socket.on("newMessage", (data) => {
```

Comment:
Thats not how this works. You're supposed to update the messages property on the App.vue component when you receive a new message.

Reply:
done here [66972bc](https://github.com/Draeggon/scratchy/pull/2/commits/66972bc3403d4e41c5a35ee175b805c6b02e7d60)

Reply:
Fixes here [1b8d540](https://github.com/Draeggon/scratchy/pull/2/commits/1b8d5404d4eef6bc36579ce3aa35dcdea88ae51d)